.github/workflows/build.yml

```diff
@@ -1,111 +1,94 @@
+name: Build
+
 on:
   workflow_call:
     inputs:
-      project-to-pack:
+      project-to-build:
         required: true
         type: string
+      project-to-test:
+        required: false
+        type: string
+      test-filter:
+        required: false
+        type: string
+        default: TestCategory!=Integration
       coverage-threshold:
         required: false
         type: number
         default: 100
-    outputs:
-      beta-version:
-        description: "Beta version number."
-        value: ${{jobs.Pack.outputs.beta-version}}
-      stable-version:
-        description: "Stable version number."
-        value: ${{jobs.Pack.outputs.stable-version}}
 
 jobs:
-  Pack:
+  Build:
     runs-on: ubuntu-latest
-    outputs:
-      beta-version: ${{steps.version.outputs.beta-version}}
-      stable-version: ${{steps.version.outputs.stable-version}}
+    env:
+      OFFICIAL_BUILD: "True"
```

Comment:
What is this for?

Reply:
It's what the reference repo was using to determine whether warnings would be treated as errors during build time.

I was debating whether or not to do the same thing, but never enforced it. I can remove it.